### ISSUE TITLE
Impaired renal tubular reabsorption of phosphate 

### ISSUE THREAD

**Issue by RichardKlaus** [obophenotype/human-phenotype-ontology#6863]
**Preferred term label:**
Impaired renal tubular reabsorption of phosphate 

**Synonyms**

**Definition (free text, please give PubMed ID)**
Decreased renal tubular reabsorption of phosphate.
PMID: 25287933, PMID: 31474092

**Parent term (use hpo.jax.org/app)**
Abnormality of renal resorption HP:0011038

**Associated term**
Renal Fanconi syndrome HP:0001994

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**
0000-0002-2234-4248

**Comment by pnrobinson** [comment]
added
